server.py

```diff
@@ -101,13 +107,13 @@ class Meta:
 
 test = User.get_or_create(username="test", password="test")
 
-q1 = Question.get_or_create(q_no=1, author=test[0])
-q2 = Question.get_or_create(q_no=2, author=test[0])
-q3 = Question.get_or_create(q_no=3, author=test[0])
-q4 = Question.get_or_create(q_no=4, author=test[0])
-q5 = Question.get_or_create(q_no=5, author=test[0])
-q6 = Question.get_or_create(q_no=6, author=test[0])
-
+q1 = Question.get_or_create(test_case_input="1", test_case_output="1", question_statement="1", author=test[0], created_date_time=datetime.datetime.now())
```

Comment:
this dummy data is starting to clutter the code now. How about we move it to another file? Now that we have a way to add questions via a HTTP request `requests.post` should be able to do it.

Let's have the flow like this:

```
python server.py &  # start this in background
python insert_dummy.py
```